.github/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Versions
+
+The latest patch version of the `2.x` release series is supported for security updates.
+
+## Reporting a Vulnerability
+
+The Requests team takes security bugs seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+**Please do not report or discuss security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Issues can be reported privately to the maintainers by opening a [Security vulnerability report](https://github.com/WordPress/Requests/security/advisories/new).
+
+Additionally, as Requests is used by the WordPress CMS, security issues can be reported via the [WordPress HackerOne](https://hackerone.com/wordpress) program.
+Full details of the WordPress Security Policy and the list of covered projects and infrastructure can be found on [HackerOne](https://hackerone.com/wordpress).
```

Comment:
Given we have a same link twice here, should we extract the actual URLs for the links into footnotes-like notation?

Reply:
Done